simtools/utils/names.py

```diff
@@ -724,6 +733,8 @@ def camera_efficiency_simtel_file_name(site, telescope_model_name, zenith_angle,
         LST-1, MST-FlashCam-D, ...
     zenith_angle: float
         Zenith angle (deg).
+    azimuth_angle: float
```

Comment:
I suggest using astropy units everywhere for zenith and azimuth. it is less prone to mistakes. Or even better make use of the `CommandLineParser.zenith_angle` and `CommandLineParser.azimuth_angle` as you did for the other file

Reply:
Changing everything to use astropy units for the file names seems a bit of overkill, no? We are also sure that these are degrees because the configurator converts to the degrees, regardless of the user input.

Reply:
Hmm, not really. It avoids confusion with the units. I would at least like to see a fixture such as:
https://github.com/gammasim/simtools/blob/44cbe1ff973a5d6e2127c916baa03a8d7720938e/simtools/utils/geometry.py#L93

but you decide if it is not necessary in this case

Reply:
It's not that simple of just adding this decorator because it means I also have to make sure I pass an astropy quantity and get the value each time it is used. I think it is overkill here for the reasons explained above.
These naming functions should move into the corresponding classes anyway actually, so maybe then we can do it.